app/controllers/messages_controller.rb

```diff
@@ -12,6 +12,8 @@ def create
     pusher_message_created(true)
     if @message.save
       pusher_message_created(false)
+      notify_all_user(@message.chat_channel.chanel_users_ids)
+      notify_mentioned_user(@mentioned_users_id)
```

Comment:
can't we use one method here? the two methods are literally doing the same thing, the only difference is `message-created` over `mentioned` as an argument to the trigger function

Reply:
Also, is there a chance that a user is present in both lists? That they get two notifications I mean

Reply:
> Also, is there a chance that a user is present in both lists? That they get two notifications I mean
Notification type is different. They will receive both but both events have different purposes.